### PR TITLE
MIG-744: MTC 1.5.0 release notes

### PR DESCRIPTION
**Pull request by apinnick**
https://issues.redhat.com/browse/MIG-744

RN for MTC 1.5.0

4.8

Preview: https://deploy-preview-33807--osdocs.netlify.app/openshift-enterprise/latest/migration-toolkit-for-containers/mtc-release-notes.html

QE approved

### CODE REVIEW

File: migration-toolkit-for-containers/mtc-1-5-release-notes.adoc [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+You can migrate from {product-title} 3.7, 3.9, 3.10, or 3.11 to {product-version} and between {product-title} 4 clusters. {mtc-short} enables you to control the migration and to minimize application downtime.
+
+[id="new-features-and-enhancements_{context}"]
+== New features and enhancements
+
+This release has the following new features and enhancements:
+
+* The *Migration resource* tree on the *Migration details* page of the web console has been enhanced with additional resources, Kubernetes events, and live status information for monitoring and debugging migrations.
+* The web console can contain up to 50 migration plans. Previously, performance was impaired if multiple migration plans were created.
```
Comment (eriknelson) [inline migration-toolkit-for-containers/mtc-1-5-release-notes.adoc:20]: > The web console can contain up to 50 migration plans.

This isn't correct, unsure where this comes from. There is no concrete limit; it's paginated.

Performance was impaired at *scale*, think 100s of plans. The way this is worded, it suggests a handful of plans impaired the performance, which is also not true.

1.5.0 is designed to support hundreds of plans, which we believe to be a likely quantity based on the default limitation of the amount of namespaces that a plan supports and the fact that many customers will likely have thousands of namespaces.

**Review by eriknelson — CHANGES_REQUESTED** [review]

File: _topic_map.yml [hunk 2]
```diff
@@ -2037,6 +2037,8 @@ Distros: openshift-enterprise,openshift-origin
 Topics:
 - Name: About MTC
   File: about-mtc
+- Name: Release notes
```
Comment (mikemckiernan) [inline _topic_map.yml:2040]: Please include the product name, project name, or some other way to identify these release notes as separate from the platform release notes.  As an example, see how [CI/CD pipelines](https://docs.openshift.com/container-platform/4.7/cicd/pipelines/op-release-notes.html) did it.

File: migrating_from_ocp_3_to_4/migrating-applications-3-4.adoc [hunk 3]
```diff
@@ -8,6 +8,16 @@ toc::[]
 
 You can migrate your applications by using the {mtc-full} ({mtc-short}) web console or from the command line.
 
+Most cluster-scoped resources are not yet handled by {mtc-short}. If your applications require cluster-scoped resources, you might have to create them manually on the target cluster.
+
+During migration, the {mtc-full} ({mtc-short}) preserves the following namespace annotations:
```
Comment (mikemckiernan) [inline migrating_from_ocp_3_to_4/migrating-applications-3-4.adoc:13]: One consideration, you can prolly get away with {mtc-short} like you did in the preceding para. Up to you.

File: migration-toolkit-for-containers/migrating-applications-with-mtc.adoc [hunk 4]
```diff
@@ -7,6 +7,16 @@ toc::[]
 
 You can migrate your applications by using the {mtc-full} ({mtc-short}) web console or from the command line.
 
+Most cluster-scoped resources are not yet handled by {mtc-short}. If your applications require cluster-scoped resources, you might have to create them manually on the target cluster.
+
+During migration, the {mtc-full} ({mtc-short}) preserves the following namespace annotations:
```
Comment (mikemckiernan) [inline migration-toolkit-for-containers/migrating-applications-with-mtc.adoc:12]: If you removed mtc-full in the previous file, friendly reminder to remove it here.

**Review by mikemckiernan — APPROVED** [review]
Looks good.  Only thought (beside qualifying that the release notes are for MTC) is that this PR merges to master whereas the platform release notes are branch specific.  I do not know the advantages and disadvantages, just sharing in case it affects which branch you want to merge to.

**Pull request merged**